integration_test/third_party_apps_test/applications/postgresql/metadata.yaml

```diff
@@ -31,6 +31,7 @@ minimum_supported_agent_version:
 supported_operating_systems: linux
 platforms_to_skip:
   # postgres is not supported on various distros.
+  - rocky-linux-cloud:rocky-linux-8-optimized-gcp-arm64
```

Comment:
The change for getting these tests to run is trivial enough that I still think we should give it a try before deferring:

```diff
--- a/integration_test/third_party_apps_test/applications/postgresql/centos_rhel/install
+++ b/integration_test/third_party_apps_test/applications/postgresql/centos_rhel/install
@@ -1,16 +1,17 @@
 set -e
 
 source /etc/os-release
+ARCH=$(uname -m)
 case "${VERSION_ID}" in
   7)
-    sudo yum -y install https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+    sudo yum -y install https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-"${ARCH}"/pgdg-redhat-repo-latest.noarch.rpm
     ;;
   8|8.*)
-    sudo yum -y install https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+    sudo yum -y install https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-"${ARCH}"/pgdg-redhat-repo-latest.noarch.rpm
     sudo yum -qy module disable postgresql
     ;;
   9|9.*)
-    sudo yum -y install https://download.postgresql.org/pub/repos/yum/reporpms/EL-9-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+    sudo yum -y install https://download.postgresql.org/pub/repos/yum/reporpms/EL-9-"${ARCH}"/pgdg-redhat-repo-latest.noarch.rpm
     ;;
   *)
     echo -n "unknown version"
```